.circleci/config.yml

```diff
@@ -423,13 +423,7 @@ workflows:
       - upload-coverage:
           requires:
             - test-cover
-      - reproducible-builds:
```

Comment:
Why are we no longer doing this only for `master`? Doesn't this job take an extremely long time (as proved by this very PR).

Reply:
I enable it unconditionally to be able to test the CI job